scripts/create_test_customer.py

```diff
@@ -58,8 +58,8 @@
 
 BASEDIR = os.path.dirname(os.path.realpath(__file__))
 DEFAULT_CONFIG = BASEDIR + "/test_customer.yaml"
-SUPPORTED_SOURCES_REAL = ["AWS", "Azure", "OCP", "GCP"]
-SUPPORTED_SOURCES = SUPPORTED_SOURCES_REAL + ["AWS-local", "Azure-local", "GCP-local"]
+SUPPORTED_SOURCES_REAL = ["AWS", "Azure", "OCP", "GCP", "IBM"]
+SUPPORTED_SOURCES = SUPPORTED_SOURCES_REAL + ["AWS-local", "Azure-local", "GCP-local", "IBM"]
```

Comment:
```suggestion
SUPPORTED_SOURCES = SUPPORTED_SOURCES_REAL + ["AWS-local", "Azure-local", "GCP-local", "IBM-local"]
```

Reply:
Thanks @myersCody - I did see I forgot to add IBM-local so I just did it. I added your suggestions as well.
Hope you will like it.
Thanks for your feedback!